cypress/integration/Admin.DonationInfo.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Admin sets donations info settings
+  As an admin
+  I want to switch the donation info on and off and like I to change to donations goal and progress
+  In order to manage the foundings
+
+  Background:
+    Given the following "users" are in the database:
+      | slug  | email             | password | id    | name      | role  | termsAndConditionsAgreedVersion |
+      | user  | user@example.org  | abcd     | user  | User-Chad | user  | 0.0.4                           |
+      | admin | admin@example.org | 1234     | admin | Admin-Man | admin | 0.0.4                           |
+    Given the following "posts" are in the database:
+      | id | title                       | pinned | createdAt  |
+      | p1 | Some other post             |        | 2020-01-21 |
+      | p2 | Houston we have a problem   | x      | 2020-01-20 |
+      | p3 | Yet another post            |        | 2020-01-19 |
```

Comment:
```suggestion
```

Why?

Reply:
Just because to have a realistic setup.
It is visible in the test environment. @ulfgebhardt